tests/system/lab/test

```diff
@@ -489,7 +489,10 @@ fi
 
 echo "======================= RANDOMWALK SETUP 2 COMPLETE ======================="
 
+# TODO: Remove the  conditional statement once FOGL-9521 is fixed
 
+VERSION_CODENAME=$(cat /etc/os-release | grep VERSION_CODENAME | cut -f2 -d"=" | tr -d '"')
+if [[ "${VERSION_CODENAME}" != "bookworm" ]]; then
```

Comment:
i2C is bookworm OS issue or Raspberry Pi 5 hardware issue?

Reply:
bookworm OS